includes/class-sensei-lesson.php

```diff
@@ -4852,6 +4903,68 @@ public function has_sensei_blocks( $lesson = null ) {
 
 		return false;
 	}
+
+	/**
+	 *
+	 * Saves the quiz post meta settings
+	 *
+	 * @param int|null $lesson_id
+	 * @param string $new_settings
+	 */
+	private function save_quiz_settings( ?int $lesson_id, array $new_settings ): void {
```

Comment:
Nullable type hints and the void return type are PHP 7.1 features. I think we are currently supporting PHP 7.0?

Reply:
Great catch @m1r0. Thanks!